sched/irq/irq_spinlock.c

```diff
@@ -85,11 +85,12 @@ static volatile uint8_t g_irq_spin_count[CONFIG_SMP_NCPUS];
 irqstate_t spin_lock_irqsave(spinlock_t *lock)
 {
   irqstate_t ret;
+  int me;
   ret = up_irq_save();
 
   if (NULL == lock)
     {
-      int me = this_cpu();
```

Comment:
So the local visibility scope is not allowed?

Reply:
@pkarashchenko 
I thought that local variables should be declared at the beginning of the function.
Is there no such rule?

Reply:
Done